src/api/spotPrice.ts

```diff
@@ -32,6 +32,8 @@ export const useSpotPrices = (
   const assets = assetsIn
     .filter((a): a is u32 | string => !!a)
     .map((a) => a.toString())
+    .filter((v, i, a) => a.indexOf(v) === i)
```

Comment:
what is for?

Reply:
okay, could we use a new Map instead? should remove duplicates and convert to string, I assume

Reply:
I think Set would be better if we need unique values, right?

Reply:
yes yes